test/data/test_functional.py

```diff
@@ -121,7 +121,7 @@ def test_basicEnglishNormalize_load_and_save(self):
 
         with self.subTest('torchscript'):
             save_path = os.path.join(self.test_dir, 'ben_torchscrip.pt')
-            ben = basic_english_normalize().to_ivalue()
+            ben = basic_english_normalize().__prepare_scriptable__()
```

Comment:
I'd interact with this via torch.jit.script. These methods are not meant to be called directly.

Reply:
See the comment above.